recipe/build.sh

```diff
@@ -13,6 +13,9 @@ set -x -e
 INCLUDE_PATH="${PREFIX}/include"
 LIBRARY_PATH="${PREFIX}/lib"
 
+# Always build PIC code for easier static linking
+CXXFLAGS="${CXXFLAGS} -O3 -fPIC"
```

Comment:
I know that Impala is compiling all of its boost with -O3 and it does not cause problems, but bears more examination